packages/clerk-js/src/core/services/authentication/AuthenticationService.ts

```diff
@@ -18,17 +18,17 @@ export class AuthenticationService {
 
   constructor(private clerk: Clerk) {}
 
-  public initAuth = (opts: InitParams): void => {
+  public async initAuth(opts: InitParams): Promise<void> {
     this.enablePolling = opts.enablePolling ?? true;
-    this.setAuthCookiesFromSession(this.clerk.session);
+    await this.setAuthCookiesFromSession(this.clerk.session);
     this.setClientUatCookieForDevelopmentInstances();
     this.clearLegacyAuthV1Cookies();
     this.startPollingForToken();
     this.refreshTokenOnVisibilityChange();
-  };
+  }
 
-  public setAuthCookiesFromSession(session: SessionResource | undefined | null): void {
-    this.updateSessionCookie(session?.lastActiveToken);
+  public async setAuthCookiesFromSession(session: SessionResource | undefined | null): Promise<void> {
+    this.updateSessionCookie(await session?.getToken());
```

Comment:
I'd like to share a bit of context. After discussing with Dimitris, we both decided that replacing `.lastActiveToken` with `getToken` makes sense because: 

1. its a more uniform way of accessing the token. The `lastActiveToken` should be considered an implementation detail of the `SessionResource` 
2. if for *any* reason, we have a valid active session that has `lastActiveToken` set to null (eg: FAPI did not include `lastActiveToken` in the response) its safer to do an extra request rather than deleted the `__session` cookie. We have not yet identified any such scenario, but we are aware of a discrepancy between the `response` and `client` props of a FAPI response: sessions in `client` include the `lastActiveToken` while sessions in `response` do not. This is just a safety measure just in case.

Reply:
Team, what's the latest update on this?